scripts/requirements-extras.txt

```diff
@@ -15,6 +15,9 @@ junit2html
 # Script used to build firmware images for NXP LPC MCUs.
 lpc_checksum
 
+# used by NXP platform to generate/flash firmware images
+spsdk == 2.3.0
```

Comment:
This can be moved to your hal, right @pdgendt ?

Reply:
It is used by an in-tree runner however, not sure what the "policy" is here.